addons/core/tests/integration/components/relative-datetime-live-test.js

```diff
@@ -0,0 +1,36 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | relative-datetime-live', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders a relative datetime', async function (assert) {
```

Comment:
Do we not always use `assert.expect` at the beginning our tests? For example, I would expect to see `assert.expect(6)` here.

Reply:
Ah.  We should.

Reply:
Fixed!